Maths/.gitignore

```
@@ -0,0 +1 @@
+/.metadata/
```

Please add unwanted files filtering to .gitignore

```suggestion
/.metadata/
*.classpath
*.project
.settings/*
*.class
```